sale_order_partner_no_autofollow/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sale Order Customer No Autofollow",
+    "summary": "Do not add customer as follower in Sales Orders",
+    "author": "Cetmix,Odoo Community Association (OCA)",
+    "version": "15.0.1.0.0",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sale",
+    "depends": ["sale"],
+    "maintainers": ["dessanhemrayev", "CetmixGitDrone"],
```

Comment:
@ivs-cetmix are these maintainer still accurate?
Did we agree to not declare bots as maintainers?

Reply:
> @ivs-cetmix are these maintainer still accurate? Did we agree to not declare bots as maintainers?

Yes, we have removed bots long ago so we don't maintain our modules any more. Looks like this remains from the old times. Please remove the "maintainer" section completely or put someone else there.

Reply:
@ivs-cetmix Okay. BTW, I still think it would be very valuable for OCA to provide so sort of mechanism to help people get notifications about changes to any addon, whether they are maintainer or not. One idea I have in that direction is https://github.com/OCA/oca-github-bot/issues/265.

Reply:
> @ivs-cetmix Okay. BTW, I still think it would be very valuable for OCA to provide so sort of mechanism to help people get notifications about changes to any addon, whether they are maintainer or not. One idea I have in that direction is [OCA/oca-github-bot#265](https://github.com/OCA/oca-github-bot/issues/265).

@sbidoul yes, totally agree with you! I think this should be a topic for a more detailed discussion. Probably during the OCA day (still hope to get there) 😄